.github/workflows/dependency-review.yaml

```diff
@@ -29,4 +30,5 @@ jobs:
       - name: 'Dependency Review'
         uses: actions/dependency-review-action@11310527b429536e263dc6cc47873e608189ba21
         with:
+          fail-on-severity: moderate
```

Comment:
Do we have cases where we have low severity issues causing problems?

Reply:
As best I can tell the teleport repo is the only repo that has this rolled out so far.  So no, but I was concerned it may cause some noise as we roll it out.  If you want to start stricter I am fine with it, I just wondered if it had been considered.